.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -5300,6 +5312,12 @@ stages:
             target: 'dd-dotnet-chiseled-arm64-smoke-tests'
             snapshotPrefix: "smoke_test"
 
+        - script: |
+            # Artifacts (log files, coredump...) produced by the container need to have their permission adjusted
+            # for upload
+            chmod -R 777 tracer/build_data/
+          displayName: Adjust permissions to upload artifacts
```

Comment:
If we want to enable this for _all_ linux snapshot tests (I think we should) we could put this at the end of `steps/run-snapshot-test.yml` (and we could put the mkdir scripts at the _Start_). That way all the apps should benefit 🤔